sdk/formrecognizer/Azure.AI.FormRecognizer/src/RawExtractedPage.cs

```diff
@@ -5,6 +5,8 @@
 
 namespace Azure.AI.FormRecognizer.Models
 {
+    /// <summary>
+    /// </summary>
```

Comment:
Paul recommended removing `language` from `RawExtractedPage` and `ExtractedLine` - they plan to remove this in v2.1.